lib/agenda/index.ts

```diff
@@ -101,7 +101,7 @@ class Agenda extends EventEmitter {
     super();
 
     this._name = config.name;
-    this._processEvery = (humanInterval(config.processEvery) ?? humanInterval('5 seconds')) as number;
+    this._processEvery = (humanInterval(config.processEvery) ?? humanInterval('5 seconds')) as number; // eslint-disable-line @typescript-eslint/non-nullable-type-assertion-style
```

Comment:
This was noted also at https://github.com/agenda/agenda/pull/361#pullrequestreview-559199264

Wasn't quite sure how to go about this either.